unabridged-works-volume-2/original/29-letters.adoc

```diff
@@ -978,15 +1055,20 @@ O fear the Lord, and watch over one another in love, faithfulness,
 and meekness of spirit; and be not of a distrustful spirit,
 but trust your father in every trial, temptation, condition, or strait.
 
+[.signed-section-closing]
 Thy friend in the truth, and for the truth`'s sake,
 
+[.signed-section-signature]
 I+++.+++ P.
 
+[.signed-section-context-close]
 27th of 11th Month, 1666
 
-TO THE SINGLE, UPRIGHT-HEARTED, AND FAITHFUL FRIENDS OF TRUTH, IN AND ABOUT THE TWO CHALFONTS
+[.letter-heading]
+To the Single, Upright-hearted, and Faithful Friends of Truth, in and about the Two CHalfonts
```

Comment:
Whoops, we got a funky `H` in `CHalfonts`

Reply:
I'm really trying to notice these before I send the PR!! Ugh!!